cluster/saltbase/salt/kubelet/default

```diff
@@ -102,5 +102,10 @@
   {% set test_args=pillar['kubelet_test_args'] %}
 {% endif -%}
 
+{% set privileged = pillar.get('allow_privileged', false)  -%}
+{% if pillar.get('e2e_test_environment', '').lower() == 'true' -%}
+  {% set privileged = "True" -%}
```

Comment:
I'm not a huge fan of turning on privileged by default for all e2e runs. We are likely to end up in a situation where we inadvertently introduce a dependency on privileged mode that can't be detected by our test infrastructure.